### PR TITLE
Update readme, prepare to release yaml_decode v1

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/dart-lang/json_serializable/issues/437

### CODE REVIEW

**Comment by kevmoo** [comment]
@natebosch – you seem the kinda guy who would want to chime in on the package name

Thoughts?

File: yaml_decode/readme.md [hunk 1]
```diff
@@ -1 +1,71 @@
 [![Pub Package](https://img.shields.io/pub/v/yaml_decode.svg)](https://pub.dartlang.org/packages/yaml_decode)
+
+`package:yaml_decode` provides a `toParsedYamlException` function to create a
+`ParsedYamlException` that contains source and error information that helps
```
Comment (jakemac53) [inline yaml_decode/readme.md]: `ParsedYamlException` from a `CheckedJsonException`

**Comment by natebosch** [comment]
It looks like the reason this exists as a separate package is because we don't want add a dependency from `json_annotation` to `yaml` or vice versa.

Can we consider changing the signature of `toParsedYamlException` such that we pass the individual fields from the `CheckedFromJsonException` instead of that type? Then this code could live in `package:yaml` right?

**Comment by kevmoo** [comment]
Going to rename the package to `checked_yaml` – see the last PR. We have a better plan here.

**Pull request closed without merging**